lang/groovy/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name            groovy
-version         2.4.11
+version         2.4.12
 
 categories      lang java
 maintainers     breun.nl:nils openmaintainer
```

Comment:
Am I correct in assuming you are the maintainer? If so, can I ask you to turn the maintainer line into
  `maintainer {breun.nl:nils @breun} openmaintainer`
the next time you modify this port?

See https://guide.macports.org/#reference.keywords.maintainers for a description of the keyword.